etc/kayobe/inventory/hosts_vars/ceph01

```diff
@@ -0,0 +1,319 @@
+---
+
```

Comment:
Did the ceph group not work for this? Is it using the kolla-ansible inventory?

I have a branch where I am fixing up ceph to be part of the storage group, which may help here?

Reply:
it was more that there was a significant diff in the contents:
```

[will@cumulus-seed hosts_vars]$ diff -Naur ceph01 ceph02
--- ceph01      2019-03-14 14:44:40.047118710 +0000
+++ ceph02      2019-03-14 14:44:40.047118710 +0000
@@ -53,16 +53,6 @@
     destination: "{{ storage_mgmt_net_name | net_ip }}"
     comment: "hint: used by 'ceph-osd'"
   - interface: "{{ storage_mgmt_net_interface }}"
-    port: 6806
-    proto: tcp
-    destination: "{{ storage_mgmt_net_name | net_ip }}"
-    comment: "hint: used by 'ceph-osd'"
-  - interface: "{{ storage_mgmt_net_interface }}"
-    port: 6807
-    proto: tcp
-    destination: "{{ storage_mgmt_net_name | net_ip }}"
-    comment: "hint: used by 'ceph-osd'"
-  - interface: "{{ storage_mgmt_net_interface }}"
     port: 6808
     proto: tcp
     destination: "{{ storage_mgmt_net_name | net_ip }}"
@@ -103,11 +93,21 @@
     destination: "{{ storage_mgmt_net_name | net_ip }}"
     comment: "hint: used by 'ceph-osd'"
   - interface: "{{ storage_mgmt_net_interface }}"
+    port: 6816
+    proto: tcp
+    destination: "{{ storage_mgmt_net_name | net_ip }}"
+    comment: "hint: used by 'ceph-osd'"
+  - interface: "{{ storage_mgmt_net_interface }}"
     port: 6817
     proto: tcp
     destination: "{{ storage_mgmt_net_name | net_ip }}"
     comment: "hint: used by 'ceph-osd'"
   - interface: "{{ storage_mgmt_net_interface }}"
+    port: 6818
+    proto: tcp
+    destination: "{{ storage_mgmt_net_name | net_ip }}"
+    comment: "hint: used by 'ceph-osd'"
+  - interface: "{{ storage_mgmt_net_interface }}"
     port: 6819
     proto: tcp
     destination: "{{ storage_mgmt_net_name | net_ip }}"
@@ -123,6 +123,11 @@
     destination: "{{ storage_mgmt_net_name | net_ip }}"
     comment: "hint: used by 'ceph-osd'"
   - interface: "{{ storage_mgmt_net_interface }}"
+    port: 6822
+    proto: tcp
+    destination: "{{ storage_mgmt_net_name | net_ip }}"
+    comment: "hint: used by 'ceph-osd'"
+  - interface: "{{ storage_mgmt_net_interface }}"
     port: 6823
     proto: tcp
     destination: "{{ storage_mgmt_net_name | net_ip }}"
@@ -147,11 +152,6 @@
     proto: tcp
     destination: "{{ storage_mgmt_net_name | net_ip }}"
     comment: "hint: used by 'ceph-osd'"
-  - interface: "{{ storage_mgmt_net_interface }}"
-    port: 6828
-    proto: tcp
-    destination: "{{ storage_mgmt_net_name | net_ip }}"
-    comment: "hint: used by 'ceph-osd'"
   - interface: "{{ storage_net_interface }}"
     port: 6789
     proto: tcp
@@ -161,7 +161,7 @@
     port: 6800
     proto: tcp
     destination: "{{ storage_net_name | net_ip }}"
-    comment: "hint: used by 'ceph-mgr'"
+    comment: "hint: used by 'ceph-osd'"
   - interface: "{{ storage_net_interface }}"
     port: 6801
     proto: tcp
@@ -193,11 +193,6 @@
     destination: "{{ storage_net_name | net_ip }}"
     comment: "hint: used by 'ceph-osd'"
   - interface: "{{ storage_net_interface }}"
-    port: 6807
-    proto: tcp
-    destination: "{{ storage_net_name | net_ip }}"
-    comment: "hint: used by 'ceph-osd'"
-  - interface: "{{ storage_net_interface }}"
     port: 6808
     proto: tcp
     destination: "{{ storage_net_name | net_ip }}"
@@ -288,12 +283,12 @@
     destination: "{{ storage_net_name | net_ip }}"
     comment: "hint: used by 'ceph-osd'"
   - interface: "{{ storage_net_interface }}"
-    port: 6827
+    port: 6826
     proto: tcp
     destination: "{{ storage_net_name | net_ip }}"
     comment: "hint: used by 'ceph-mds'"
   - interface: "{{ storage_net_interface }}"
-    port: 6829
+    port: 6827
     proto: tcp
     destination: "{{ storage_net_name | net_ip }}"
     comment: "hint: used by 'ceph-osd'"
```

Reply:
oh... interesting. how nuts. I had no idea that is what ceph-ansible had done